.github/scripts/toolchain/build-mingw.sh

```diff
@@ -56,15 +56,23 @@ if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "$MINGW_BUILD_PATH/Makefile" ]]; then
             ;;
         esac
 
+        case "$ARCH-$PLATFORM" in
+            x86_64-w64-mingw32)
+                LDFLAGS="$LDFLAGS -L$TOOLCHAIN_PATH/lib/gcc/$TARGET/lib"
```

Comment:
I don't know why but without this, there is issue that linker cannot find `libgcc_s` on `x86_64-w64-mingw32`. This is not the case for `aarch64-w64-mingw32`.